fuse/index.d.ts

```diff
@@ -1,30 +1,36 @@
 // Type definitions for Fuse.js 2.5.0
 // Project: https://github.com/krisk/Fuse
-// Definitions by: Greg Smith <https://github.com/smrq/>
+// Definitions by: Alan Agius <https://github.com/alan-agius4/>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare class Fuse {
-	constructor(list: any[], options?: fuse.IFuseOptions);
-	search(pattern: string): any[];
-}
+declare module 'fuse.js' {
+	namespace fuse.js {
```

Comment:
It would be better to leave the namespace as `fuse` or `fuse.js`, because as it is this unnecessarily creates a `fuse` namespace with an inner `js` namespace.